index.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
-var mkdirp = require('mkdirp');
-var md5Hex = require('md5-hex');
 var fs = require('fs');
 var path = require('path');
+var mkdirp = require('mkdirp');
+var md5Hex = require('md5-hex');
 var writeFileAtomic = require('write-file-atomic');
+var packageHash = require('package-hash');
```

Comment:
I like the idea of `package-hash` (a lot!). My one concern is that using it so prolifically in AVA and it's dependencies is that all these file system calls will add up.

`require.resolve` alone definitely has performance impacts: https://www.npmjs.com/package/cache-require-paths.

 `package-hash` causes a `resolve`, a `git` call, plus hashing. I would love to see some benchmarks to understand exactly how much it costs to `package-hash` a module.